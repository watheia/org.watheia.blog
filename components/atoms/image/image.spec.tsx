import * as React from "react"
import { render } from "@testing-library/react"
import Image from "."

describe("ui.atoms.Image", () => {
  it("should be rendered correctly", () => {
    const { asFragment } = render(<Image src="https://cdn.watheia.org/assets/icon.png" />)
    expect(asFragment()).toMatchSnapshot()
  })
})
