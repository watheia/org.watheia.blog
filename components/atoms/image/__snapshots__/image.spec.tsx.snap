// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui.atoms.Image should be rendered correctly 1`] = `
<DocumentFragment>
  <span
    class=" lazy-load-image-background opacity"
    style="display: inline-block; height: auto; width: auto;"
  >
    <img
      alt="..."
      class="image makeStyles-root-1 makeStyles-dBlock-2"
      height="auto"
      src="https://cdn.watheia.org/assets/icon.png"
      width="auto"
    />
  </span>
</DocumentFragment>
`;
